indra/sources/phosphoelm/processor.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import requests
+
+from indra.statements import Phosphorylation, Evidence, Agent
+
+from .phosphoelm_mapping import phosphoelm_mapping
+
+gilda_url = 'http://grounding.indra.bio/ground'
+logger = logging.getLogger(__name__)
+
+
+def _gilda_grounder(entity_str):
+    # Try to find a namespace for the enzyme entity string, return the
+    # string that provided the match
+    res = requests.post(gilda_url, json={'text': entity_str})
+    if res.status_code == 200 and res.json():
+        db_ns = res.json()[0]['term']['db']
+        db_id = res.json()[0]['term']['id']
+        return entity_str, db_ns, db_id
+    else:
+        if res.status_code != 200:
+            logger.warning('Gilda service responded with status code %d' %
+                           res.status_code)
+        return entity_str, None, None
+
+
+class PhosphoElmProcessor(object):
+    """Processes data dumps from the phospho.ELM database.
+
+    See http://phospho.elm.eu.org/dataset.html
+
+    Parameters
+    ----------
+    phosphoelm_data : list(dict)
+        JSON compatible list of entries from a phospho.ELM data dump
+
+    Attributes
+    ----------
+    statements : list(indra.statement.Phosphorylation)
+        A list of the phosphorylation statements produced by the entries
+        in phosphoelm_data
+    """
+    def __init__(self, phosphoelm_data=None):
```

Comment:
I don't think the `phosphoelm_data` argument should be optional - the processor is not meaningful to instantiate without something to process. Consequently, the 
```
        if self._phosphoelm_data is None:
            logger.warning('No phosphoelm data to process. No statements '
                           'are produced')
            return
```
block below can also be removed.